bindata/oauth-apiserver/ns.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/node-selector: ""
+  name: openshift-oauth-apiserver
+  labels:
+    openshift.io/run-level: "1"
```

Comment:
drop this line

Reply:
we want run with SCCs. No need for a runlevel.

Reply:
interesting didn't know that `run-level` prevents assigning `SCC` to a `pod`